README.md

```diff
@@ -6,12 +6,12 @@ The Student Robotics public website.
 
 ## Getting Started
 
-1. [Install Ruby][install-ruby]
+1. [Install Ruby 2.7][install-ruby]
```

Comment:
Do we care that it's 2.7 specifically, or would a later version be ok?
(Might be useful to record somewhere, though perhaps not in the setup instructions) why a specific version is needed)

Reply:
[GitHub Pages uses 2.7 specifically.](https://pages.github.com/versions/) We probably could use 3.0 but I think it makes sense to stick to what is used on GitHub Pages.